templates/inscription.html

```diff
@@ -23,6 +23,8 @@ <h1>Inscription {{ self.number }} (unstable)</h1>
   <dt>parent</dt>
   <dd><a class=monospace href=/inscription/{{ parent }}>{{ parent }}</a></dd>
 %% }
+  <dt>sequence number</dt>
+  <dd>{{ self.sequence_number }}</dd>
```

Comment:
I think that people don't want to add a sequence number below a certain activation number to keep the original numbers prominent socially. What if we didn't show the sequence number below a threshold and instead just showed that if you run the API? We would still index both but we would only show inscription number for earlier inscriptions.
```html
%% if self.inscription_number >= 50,000,000 {
<dt>sequence number</dt>
  <dd>{{ self.sequence_number }}</dd>
  }
```

Reply:
Another idea maybe we think of this like an engineering tolerance.
```html
%% if self.inscription_number >= 50,000,000 {
<dt>sequence number</dt>
  <dd>+ {{ self.sequence_number - self.inscription_number }}</dd>
  }
  elseif self.inscription_number < 50,000,000 &&  self.inscription_number > 0{
  <dt>sequence offset</dt>
  <dd>+ {{ self.sequence_number - self.inscription_number }}</dd>
  }
  elseif self.inscription_number < 0{ 
    <dt>sequence number</dt>
  <dd>+ {{ self.sequence_number }}</dd>
  }
  
  ```

Reply:
We specifically want to display sequence numbers so that people understand exactly how inscription numbers would change *if* they were made unstable. Right now people are pretty much in the dark.

Reply:
The sentiment I found on all the twitter spaces is people don't want to change the prior inscription numbers at all behind a certain activation number. Their view is the trust in the entire protocol will be broken if we decide to change these numbers and make them unstable and we changed to a sequence number. 

I think the solution that a lot of people are in agreement with on both sides is @huuep comment:
https://github.com/ordinals/ord/discussions/2458#discussioncomment-7051777
He proposes that we immutably store the prior inscription numbers using an inscription. Then at a certain inscription number activation we will then move forward with immutable sequence numbers. 

I tend to agree with you. However I want to make sure the other voices are heard here.

Reply:
yeah gonna remove it from the front-end